src/main/webapp/components/StatusBox.js

```diff
@@ -19,6 +24,10 @@ export const StatusBox = hh(class StatusBox extends Component {
       attachmentsApproved = ''
     } = this.props.status;
 
+    if (type !== '' && projectKey !== '') {
```

Comment:
looks like spinner makes no sense on this component. Please remove or justify its use.

Reply:
It is another good candidate for a functional component.

Reply:
Removed, I agree on making a functional component.